extensions/agent/src/data/pickScheduleData.ts

```diff
@@ -20,12 +20,18 @@ export class PickScheduleData implements IAgentDialogData {
 		this.jobName = jobName;
 	}
 
-	public async initialize() {
+	public async initialize(): Promise<azdata.AgentJobScheduleInfo[]> {
 		let agentService = await AgentUtils.getAgentService();
-		let result = await agentService.getJobSchedules(this.ownerUri);
-		if (result && result.success) {
-			this.schedules = result.schedules;
+		try {
+			let result = await agentService.getJobSchedules(this.ownerUri);
```

Comment:
I wouldn't bother wrapping this in a try/catch - it's just going to hide the original source of the error if an exception is thrown